<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelRegistration</class>
 <widget class="QMainWindow" name="PanelRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Номер</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="number_spinBox"/>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>IdCard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="idCard_spinBox"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>1</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Фомилия Имя Отчество</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Пол</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Спортивный разряд</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="name_1"/>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="date_1"/>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="fluring_1">
         <item>
          <property name="text">
           <string>Муж</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Жен</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="sportsCategory_1">
         <item>
          <property name="text">
           <string>б/р</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>КМС</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>МС</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
      <zorder>label_4</zorder>
      <zorder>fluring_1</zorder>
      <zorder>label</zorder>
      <zorder>date_1</zorder>
      <zorder>label_2</zorder>
      <zorder>name_1</zorder>
      <zorder>label_3</zorder>
      <zorder>sportsCategory_1</zorder>
      <zorder>label_4</zorder>
      <zorder>fluring_1</zorder>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>date_1</zorder>
      <zorder>sportsCategory_1</zorder>
      <zorder>label_3</zorder>
      <zorder>name_1</zorder>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Фомилия Имя Отчество</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Пол</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Спортивный разряд</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="name_2"/>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="date_2"/>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="fluring_2">
         <item>
          <property name="text">
           <string>Муж</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Жен</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="sportsCategory_2">
         <item>
          <property name="text">
           <string>б/р</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>КМС</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>МС</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>3</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Фомилия Имя Отчество</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Пол</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Спортивный разряд</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="name_3"/>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="date_3"/>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="fluring_3">
         <item>
          <property name="text">
           <string>Муж</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Жен</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="sportsCategory_3">
         <item>
          <property name="text">
           <string>б/р</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>КМС</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>МС</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>4</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Фомилия Имя Отчество</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Пол</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Спортивный разряд</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="name_4"/>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="date_4"/>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="fluring_4">
         <item>
          <property name="text">
           <string>Муж</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Жен</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="sportsCategory_4">
         <item>
          <property name="text">
           <string>б/р</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1ю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>КМС</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>МС</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Делегация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="delegation_line"/>
      </item>
     </layout>
    </item>
    <item row="6" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Теретория</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="territory_line"/>
      </item>
     </layout>
    </item>
    <item row="7" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Предстовитель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="representative_line"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="8">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
   <zorder>tableView</zorder>
   <zorder>number_spinBox</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>idCard_spinBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>delegation_line</zorder>
   <zorder>territory_line</zorder>
   <zorder>representative_line</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder></zorder>
   <zorder>label_9</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
